extension/src/popup/views/SignTransaction/index.tsx

```diff
@@ -234,126 +259,130 @@ export const SignTransaction = () => {
       customSubmit={verifyPasswordThenSign}
     />
   ) : (
-    <div className="SignTransaction">
-      <ModalWrapper>
-        <ModalHeader>
-          <strong>{t("Confirm Transaction")}</strong>
-        </ModalHeader>
-        {flaggedKeyValues.length ? (
-          <FlaggedWarningMessage
-            isUnsafe={isUnsafe}
-            isMalicious={isMalicious}
-            isMemoRequired={isMemoRequired}
-          />
-        ) : null}
-        {!isDomainListedAllowed && !isSubmitDisabled ? (
-          <FirstTimeWarningMessage />
-        ) : null}
-        <div className="SignTransaction__info">
-          <Card variant={Card.variant.highlight}>
-            <PunycodedDomain domain={domain} isRow />
-            <div className="SignTransaction__subject">
-              {t("is requesting approval to a")} {isFeeBump ? "fee bump " : ""}
-              {t("transaction")}:
-            </div>
-            <div className="SignTransaction__approval">
-              <div className="SignTransaction__approval__title">
-                {t("Approve using")}:
+    <>
```

Comment:
the only change I made in the jsx is adding the `{hwStatus === HwOverlayStatus.IN_PROGRESS && <LedgerSign />}`, the diff is a little deceiving